src/Features/LanguageServer/ProtocolUnitTests/References/FindAllReferencesHandlerTests.cs

```diff
@@ -52,6 +53,51 @@ void M2()
             AssertValidDefinitionProperties(results, 0, Glyph.FieldPublic);
         }
 
+        [WpfFact(Skip = "https://github.com/dotnet/roslyn/issues/43063")]
+        public async Task TestFindAllReferencesAsync_Streaming()
```

Comment:
should this be skipped?  looks like a new test.

Reply:
I ended up leaving this as skipped because I don't know why they fail in CI (the link in the issue 404s), so I have no expectation that this test will be any different.

Reply:
I seem to recall it was skipped due to streaming issues.  But since the link no longer works, we won't be able to diagnose it.  The handler as well has been through a bunch of issues since the issue was filed, so it may not even be applicable.  If these are running stably for you, I think we should just unskip all these.  At worst we have to disable them again, but we'll have updated diagnostic information.

Reply:
> If these are running stably for you, I think we should just unskip all these. At worst we have to disable them again

Yep, thats what I did yesterday. They failed, and in the non-streaming case: https://dev.azure.com/dnceng/public/_build/results?buildId=858041&view=ms.vss-test-web.build-test-results-tab&runId=27467314&paneView=debug&resultId=165225

Reply:
Ah interesting.  May want to leave it skipped then for now .  Can you at least add the failure to the issue so we have an up to date failure?  We should probably try and figure out what is going on and unskip these.

Reply:
Good idea, I probably should have thought of that :)

Reply:
I think the tests were originally disabled because the `results` array reported the data in a non-deterministic order (which is probably something I should go back and look into soon).